### DISCUSSION TITLE
Unable to run app (vulkan.hpp not found)

### DISCUSSION THREAD

**Discussion by ErAzOr2k** [shogo4405/HaishinKit.kt#311] [Q&A]
After build the app in Android Studio I get this error:
fatal error: 'vulkan/vulkan.hpp' file not found
#include <vulkan/vulkan.hpp>

What can I do? :D

**Comment by shogo4405** [comment]
Try following these steps.
- https://github.com/shogo4405/HaishinKit.kt?tab=readme-ov-file#-examples
